fiaas_skipper/__init__.py

```diff
@@ -6,6 +6,9 @@
 import logging
 import os
 
+from gevent import monkey
+monkey.patch_all()  # NOQA
```

Comment:
I'm not sure if it actually matters, but maybe this should happen before the `json`, `logging` and `os` imports also?